### ISSUE TITLE
Deprecate function in use

### ISSUE THREAD

**Issue by ilovelinux** [wpscanteam/wpscan#1189]
```
$ wpscan ...
NOTE: Gem.gunzip is deprecated; use Gem::Util.gunzip instead. It will be removed on or after 2018-12-01.
....
````

**Comment by ethicalhack3r** [comment]
Hey! I don't think we use the ```gunzip``` gem in wpscan... maybe the warning is coming from another project?

**Comment by ilovelinux** [comment]
Hmm you're right.
I forgot to post a line. Here's the full log:
```
$ wpscan ...
NOTE: Gem.gunzip is deprecated; use Gem::Util.gunzip instead. It will be removed on or after 2018-12-01.
Gem.gunzip called from /opt/wpscan/vendor/bundle/ruby/2.5.0/gems/unicode-display_width-1.3.0/lib/unicode/display_width/index.rb:5.
```

**Comment by FireFart** [comment]
Just run bundle update to install the latest gem (or update using your package manager, depends on how you installed wpscan). It was fixed in version 1.3.3:
https://github.com/janlelis/unicode-display_width/blob/master/CHANGELOG.md
